Mono.Addins/Mono.Addins.Description/AddinDescription.cs

```diff
@@ -526,7 +528,26 @@ internal bool SupportsVersion (string ver)
 				return conditionTypes;
 			}
 		}
-		
+
+		/// <summary>
+		/// Gets the localizer types.
+		/// </summary>
+		/// <value>
+		/// The localizer types.
+		/// </value>
+		public LocalizerTypeDescriptionCollection LocalizerTypes {
+			get {
+				if (localizerTypes == null) {
+					localizerTypes = new LocalizerTypeDescriptionCollection (this);
+					if (RootElement != null) {
+						foreach (XmlElement elem in RootElement.SelectNodes ("LocalizerType"))
+							localizerTypes.Add (new LocalizerTypeDescription (elem));
+					}
+				}
+				return localizerTypes;
```

Comment:
This should actually be a collection of ExtensionNodeDescription. The goal is to be able to fully specify a localizer (for example, gettext with some catalog location), and then reference that localizer from an add-in. The properties of a localizer depend on the type of the localizer, so we need to store the whole node, like we do for the Localizer property.

Reply:
Done!